api/app/Models/User.php

```diff
@@ -1171,6 +1178,12 @@ public function getImproveBehaviouralSkillsRankingAttribute()
             ->sortBy('improve_skills_rank');
     }
 
+    public function scopeIsVerifiedGovEmployee(Builder $query): void
+    {
+        $query->where('computed_is_gov_employee', true)
+            ->whereNotNull('work_email_verified_at');
```

Comment:
It's probably not possible to set `work_email_verified_at` without setting a work email so this probably works fine.  However, our definition includes `work_email` is not null and I think it is more understandable and secure if that's included here explicitly.

Reply:
I agree, that makes sense to me! Added in [add work email to verified employee scope](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12847/commits/8689dad7492d46f5cbf1ae05b5f9063c62377c6b)